packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.js

```diff
@@ -101,6 +101,12 @@ export default class QuanticTab extends LightningElement {
   }
 
   get tabClass() {
-    return `slds-tabs_default__item ${this.isActive ? 'slds-is-active' : ''}`;
+    return `slds-button slds-tabs_default__item tab_button ${
+      this.isActive ? 'slds-is-active' : ''
+    }`;
+  }
+
+  get isPressed() {
```

Comment:
I don't think you need that function anymore.

Reply:
Comments added for function isPressed.